server/command_hooks.go

```diff
@@ -17,10 +17,10 @@ func (p *Plugin) registerCommand(teamId string) error {
 		TeamId:           teamId,
 		Trigger:          CommandTrigger,
 		AutoComplete:     true,
-		AutoCompleteHint: "(true|false)",
-		AutoCompleteDesc: "Enables or disables the demo plugin hooks.",
+		AutoCompleteHint: "(true|false|ephemeral)",
+		AutoCompleteDesc: "Enables or disables the demo plugin hooks, or demonstrates an ephemeral post capabilities.",
 		DisplayName:      "Demo Plugin Command",
-		Description:      "A command used to enable or disable the demo plugin hooks.",
+		Description:      "A command used to enable or disable the demo plugin hooks, or demonstrate ephemeral post capabilities.",
```

Comment:
I would like to add this as a new command. 3/5
Maybe just `/ephemeral`? 1/5

Reply:
Created a new command, and felt `demo_ephemeral` was a better candidate.  Open to changes.

Reply:
In #32 `/crash` is used and it quite like this. I'm 3/5 on `/ephemeral` now. It's a short and clear name for what the plugin does. Why do you want to go with `demo_ephemeral`?

Reply:
I had originally wanted to preserve the 'demo_*' namespace, but if we are going to add `/crash` it makes sense to use `/ephemeral` too.